src/PIL/Image.py

```diff
@@ -355,10 +355,11 @@ def init():
     if _initialized >= 2:
         return 0
 
+    parent_module, _, _ = __name__.rpartition(".")
```

Comment:
```suggestion
    parent_name = __name__.rpartition(".")[0]
```
Actually, looking at https://docs.python.org/3/library/importlib.html#approximating-importlib-import-module, I wonder if `parent_name` would be better?

Reply:
Good idea. I've replaced it